conventional_pre_commit/format.py

```diff
@@ -23,7 +23,7 @@ def r_types(types):
 
 def r_scope():
     """Regex str for an optional (scope)."""
-    return r"(\([\w \/-]+\))?"
+    return r"(\([\w \/-:]+\))?"
```

Comment:
Can you please swap the `-` and the `:` in this piece?

Placing the colon after the hyphen causes this part of the expression to be considered a range, e.g. any character between `/` and `:` ([ASCII code 47 - 58](https://www.ascii-code.com/)).

Now this isn't really a problem since that range in ASCII represents the numbers (already allowed by the `\w`). But it isn't what was intended here I think.

Reply:
Ok, sorry. Didn't have this considered. 
So, here comes the change. :)